src/components/Table/GridTable.stories.tsx

```diff
@@ -1289,3 +1289,29 @@ export function SelectableRows() {
     </>
   );
 }
+
+export function RevealOnRowHover() {
+  const nameColumn: GridColumn<Row> = {
+    header: "Name",
+    data: ({ name }) => ({ content: name }),
+    mw: "160px",
+  };
+
+  const revealColumn: GridColumn<Row> = {
+    header: "Reveal",
+    data: ({ value }) => ({ content: <span>{value}</span>, revealOnRowHover: true }),
```

Comment:
**question:** I am curious to know if this works when the `content` does not return an element. For instance, if this works with:
```
data: ({ value }) => ({ content: value, revealOnRowHover: true })
```
I'm thinking it might not based on the CSS selector. We may need to tell the user as a JSDoc above the `revealOnRowHover` prop that the content of the cell needs to be an element in order to work.

Reply:
let me give it a try!

Reply:
hmm I got the same results as we saw during vc with the different tests we tried

Reply:
Which was what - that the cell's content value **isn't** hidden? I am guessing that would be the case because our CSS selector is not hiding the cell, but any direct children within the cell. I was curious if a direct value, not wrapped in an element, would hide or not. But sounds like it doesn't hide, which is kind of expected as there are no elements nested in the cell, just `textContent`.

Reply:
Based on that, I'd suggest updated the JS Doc for the `revealOnRowHover` to specify that it requires the content of the cell to be wrapped in an element to be hidden.